apps/arena/lib/arena/game/player.ex

```diff
@@ -279,9 +279,9 @@ defmodule Arena.Game.Player do
   ####################
   # Internal helpers #
   ####################
-  defp skill_key_execution_action(skill_key) do
-    "EXECUTING_SKILL_#{String.upcase(skill_key)}" |> String.to_existing_atom()
-  end
+  defp skill_key_execution_action("1"), do: :EXECUTING_SKILL_1
```

Comment:
These changes are already included in #367 🤔

Reply:
Yeah but I need it for the mocked players' actions. Otherwise the app crashes when game starts.

Reply:
Shouldn't this PR be pointed towards that one then? Either that, or we close #367 when we merge this

Reply:
Let's see which gets merged first 👀

Reply:
#367 won